lib/spack/spack/binary_distribution.py

```diff
@@ -676,6 +676,8 @@ def get_specs(force=False, use_arch=False, names=[]):
     if use_arch:
         arch_pattern = '(%s-%s-[^-]*)' % (arch.platform, arch.os)
 
+    if names is None:
+        names = ['']
```

Comment:
In this PR we changed the default value from an empty list to a list with an empty string in it. Just pointing it out in case this makes a difference and results from a typo and not a deliberate choice.

Reply:
Need something to put into the regular expression.